src/library/autodj/autodjprocessor.cpp

```diff
@@ -1665,6 +1669,26 @@ void AutoDJProcessor::playerRateChanged(DeckAttributes* pAttributes) {
     calculateTransition(fromDeck, getOtherDeck(fromDeck), false);
 }
 
+void AutoDJProcessor::playlistFirstTrackChanged() {
+    qDebug() << this << "playlistFirstTrackChanged";
+    if (m_eState != ADJ_DISABLED) {
+        DeckAttributes* pLeftDeck = getLeftDeck();
+        DeckAttributes* pRightDeck = getRightDeck();
+        // if (!pLeftDeck || !pRightDeck) {
+        //     // User has changed the orientation, disable Auto DJ
+        //     toggleAutoDJ(false);
+        //     emit autoDJError(ADJ_NOT_TWO_DECKS);
+        //     return ADJ_NOT_TWO_DECKS;
+        // }
```

Comment:
I just don't know what to do with these commented lines as I don't know what deck orientation is, and how user can change it.